src/config/default_config.yaml

```diff
@@ -112,5 +112,9 @@ map:
   findSymbol: "CTRL+T"
   aRestart: "CTRL+F4"
 
+rr_backend_enabled: false
```

Comment:
@Madman10K had the idea to change our configs to have more feature-like mappings: e.g. here this would be IMO lilke:

``yaml
rr_backend:
  enabled: ..
  path: ..
```